test/Microsoft.AspNet.Mvc.Core.Test/TempDataDictionaryTest.cs

```diff
@@ -0,0 +1,228 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.Framework.Internal;
+using Moq;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Core
+{
+    public class TempDataDictionaryTest
```

Comment:
Are there old tests to port? Is this a port of them? Hint, this is a good idea if that's the case.

Also, `Microsoft.AspNet.Mvc`